radicle/src/identity/did.rs

```diff
@@ -46,6 +47,18 @@ impl From<crypto::PublicKey> for Did {
     }
 }
 
+impl From<Author> for Did {
+    fn from(key: Author) -> Self {
+        (key.id()).into()
+    }
+}
+
+impl From<&Author> for Did {
+    fn from(key: &Author) -> Self {
+        Self(*key.id())
+    }
+}
```

Comment:
Organisational nit: I'd expect these implementations to live beside `Author`, since `Did` is more fundamental than `cob::Author`.

Reply:
Has been reverted, uses what is implemented in #324